drivers/SmartThings/matter-switch/profiles/electrical-power.yml

```diff
@@ -0,0 +1,14 @@
+name: electrical-power
+components:
+- id: main
+  capabilities:
+  - id: switch
```

Comment:
Let's define a naming convention for profiles for electrical sensor devices. I think in most cases, we would expect to see the electrical sensor on a plug, so if we include the switch capability and the `SmartPlug` category, what if we started the profile name with `plug`? There isn't a convention yet, so I think we should start one in order to make profile name building in the future easier if we need to expand on it. Maybe something like this, where we start with a switch/plug type depending on the device type and that is used to match to the correct category in a profile, and then the following additions are optional checks based on the device functionality which we would append in this order:
switch/plug-(electrical)-(energy)-(power)-(powerConsumption)

What do you think about establishing this convention?

Reply:
I like that idea! First off, I think we can scrap the `electrical` keyword since that doesn't really map to any capabilities we'd want to specify. Second, the proposed convention would already by broken a little with the eve profile called `power-energy-powerConsumption` , which using this naming convention should be called `plug-energy-power-powerConsumption`. Personally, I propose we make a second profile with the new name and deprecate the old one, especially since the eve-specific functionality may merge with the electrical sensor functionality someday. Further, since we got some backing for allowing these one-time driver updates, I think this is exactly the kind of thing we're looking to be able to re-organize if we want. 

But other than this little snag, I like this. What do you think about making a new profile to fit the new naming convention, and then just sticking by it?

Reply:
We can revert it if we want, but for now I just bit the bullet and made a new profile with the same functionality as already existed in the Eve-specific profile. I haven't touched the Eve stuff though, only this new stuff.